controllers/ldapidentityprovider/controller.go

```diff
@@ -102,6 +102,8 @@ func (r *Controller) ReconcileDelete(req controllers.Request) (ctrl.Result, erro
 }
 
 // SetupWithManager sets up the controller with the Manager.
+//
+//nolint:wrapcheck
```

Comment:
Ditto for this one; why disable wrapcheck?

Reply:
See above.  Same reason.